README.md

```diff
@@ -35,6 +35,15 @@ from the [Releases](https://github.com/gladly/app-platform-appcfg/releases) sect
 | Windows 64 bit Intel | appcfg-{version number}-win.zip |
 | Linux 64 bit Intel | appcfg-{version number}-linux-intel.zip |
 
+Once downloaded, open the zipped file, and place the executable appcfg in `~/.local/bin/`.
```

Comment:
I would recommend that on Mac OS X and Linux the files should be copied to `/usr/local/bin` which should already be be in the path.

We also need instructions for Windows.